Constitution.md

```diff
@@ -145,43 +141,65 @@ If there is more then 1 president, they exist as a single voting member on counc
 3. Monitor maintenance of the CSSA Office, CSSA Lounge, and purchase pertinent supplies when needed.
 4. Ensure appropriate reservation, organization, and signage for events that take place in the CSSA Lounge.
 5. Ensure that the CSSA lounge is a safe and inclusive space.
-6. Lead the CSSA lounge supervisors and work with council in appointing them.
-7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+6. Work with the Director of Finances and the Finance Committee to record lounge transactions.
+7. Lead the Lounge Committee, call committee meetings and delegate tasks to committee members.
+8. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
 
 ### Director of Student Affairs
 
 1. Act as a liaison for the students towards the Department of Computer Science, Faculty of Science and UMSU.
 2. Act as a liaison between students and Council.
-3. Stay up to date on departmental and Senate meetings pertaining to the Computer Science program.
-4. Lead relevant campaigns on issues affecting the Computer Science Student Community.
-5. Work with the Director of Advocacy to promote opportunities of initiative/programs/organizations pertaining to where the Computer Science community is underrepresented.
-6. Ensure that the CSSA lounge is a safe and inclusive space.
-7. Lead the Outreach Committee with up to 3 committee members.
-8. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+3. Stay up to date on Faculty and Senate meetings pertaining to the Computer Science program.
+4. Attend Student Executive Assembly (SEA) Meetings and Department of Computer Science Meetings on behalf of the CSSA, or put fair effort into acquiring a proxy from Council.
+5. Summarize the relevant contents of external meetings, including but not limited to those listed above, to Council.
+6. Lead relevant campaigns on issues affecting the Computer Science Student Community.
```

Comment:
nit:

```suggestion
6. Lead relevant campaigns on issues affecting computer science students.
```

unless Computer Science Student Community is an actually defined group somewhere.

Reply:
Maybe a more accurate would be parts of the CS community that are affected by it? Or is this like a united we stand so if it affects some we all go to war

Reply:
Agree that "Computer Science Student Community" sounds like an official thing the way it's layed out here. I kinda prefer "computer science students"